org.eclipse.paho.client.mqttv3/src/main/java/org/eclipse/paho/client/mqttv3/internal/websocket/WebSocketSecureNetworkModule.java

```diff
@@ -39,6 +39,7 @@
 	private String uri;
 	private String host;
 	private int port;
+	private Properties customWebSocketHeaders;
```

Comment:
And why is it a Properties field now, instead of a Map? Properties is deprecated and the reading/writing abilities of Properties are not used in this case? Can I issue a PR to change this to Map please?

Reply:
Properties is used for sslClientProps. 
Sure, Map is better. 
New pull request - https://github.com/eclipse/paho.mqtt.java/pull/555